dipy/reconst/sfm.py

```diff
@@ -39,7 +39,7 @@
 # If sklearn is unavailable, we can fall back on nnls (but we also warn the
 # user that we are about to do that):
 if not has_sklearn:
-    w = "sklearn is not available, you can use 'nnls' method to fit"
+    w = sklearn._msg + "\nAlternatively, you can use 'nnls' method to fit"
```

Comment:
Same here. I must be missing something. Where is `sklearn._msg` defined?

Reply:
See my answer in https://github.com/dipy/dipy/pull/2480#discussion_r743658093.

Reply:
Got it. Yes, and the line that defines this variable is https://github.com/dipy/dipy/blob/c22e18df734e1f410728dbe5d779f19406285e37/dipy/utils/tripwire.py#L46

Reply:
Right. Resolving then.